recipes/opensartoolkit/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "opensartoolkit" %}
+{% set version = "0.12.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opensartoolkit-{{ version }}.tar.gz
+  sha256: 867feb004333beb8245cbb6c548753bdc4d35545b99ee4496013a830442ea145
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
```

Comment:
```suggestion
    - python >=3.6
    - setuptools
```

Reply:
Try accept this suggestions, let's see if it pushes a commit. If not, you can just close this PR and start a new one...